Testing/functional/tests/cypress/integration/ui/user/waitlist.js

```diff
@@ -30,6 +30,11 @@ function getTicket(status, queuePosition = 0) {
 describe("Waitlist Ticket Module Enabled", () => {
     beforeEach(() => {
         cy.logout();
+        cy.configureSettings({
+            waitingQueue: {
+                enabled: true,
+            },
+        });
         cy.enableModules(["Ticket"]);
```

Comment:
The enableModules call should be rolled into configureSettings.

Reply:
Fixed in AB#14995